proposals/exception-handling/Exceptions-formal-overview.md

```diff
@@ -222,6 +223,11 @@ S;C, labels (catch [t*]) ⊢ instr* : []→[t*]
 S;C, labels [t*] ⊢ caught{a val^n} instr* end : []→[t*]
 ```
 
-## TODO Uncaught Exceptions
+## Uncaught Exceptions
+
+To describe uncaught exceptions we add a new terminal value.
+
+```
+result ::= ... | T[val* (throw tagaddr)]
```

Comment:
What `result` are we talking about? Can we expand `...` if that's not too long?

Reply:
This refers to [these result values](https://webassembly.github.io/spec/core/exec/runtime.html#syntax-result). I now expanded `...` and added this link in the document as well.